### ISSUE TITLE
DOC: User's Guide needs section for contouring and pcolor-like plots

### ISSUE THREAD

**Issue by efiring** [matplotlib/matplotlib#7116]
This is prompted by #7103, but the documentation gap has been obvious for many years.

**Comment by soma2000-lang** [comment]
@efiring  Can I work on this??

**Comment by efiring** [comment]
Certainly! The structure and content of the docs have changed quite a bit since I opened this issue (which I completely forgot about).  I suggest that you start with a rough outline to specify the information you would like to add, and what section or sections it would go in, given the current structure of the docs. I'm guessing it would mainly be in one or more tutorials.

**Comment by jklymak** [comment]
I think this is still sort of a gap.  Of course we have added [plot types](https://matplotlib.org/stable/plot_types/index.html), a   [section](https://matplotlib.org/stable/users/explain/quick_start.html#color-mapped-data) to the Quick Start Guide, and the Axes documentation has sections for [contour](https://matplotlib.org/stable/api/axes_api.html#contours) and [2d arrays](https://matplotlib.org/stable/api/axes_api.html#d-arrays).  

I think the [User Guide](https://matplotlib.org/stable/users/index.html) could have a major section, between "Axes" and "Artists" that has the main visualization types broken out and introduced relatively carefully.  Probably should follow a similar order to the Axes API reference.

**Comment by sahil-11116** [comment]
Hi I'm a new contributor to Matplotlib and would like to take this documentation issue. I'd appreciate any guidance or suggestions on where to begin. Thank you.

**Comment by timhoffm** [comment]
@sahil-11116  thanks for your interest. Discussion on the topic has moved on meanwhile. This is a special case of the larger topic on how to present and teach plotting functions in the user guide, which is discussed in detail in #31496 and #29124.

I claim that the linked issue and PR cover this issue and therefore close here.
